db/models/review.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Review = db.model({
+	body: {
+		type: Sequelize.TEXT,
+		allowNull: false,
+	}, 
+	rating: {
+		type: Sequelize.INTEGER.UNSIGNED,
```

Comment:
i googled UNSIGNED because i didn't know what it was, and it seems Postgres doesn't support it. but having min and max validate should make up for that anyway.